services/QuillLMS/config/initializers/secure_headers.rb

```diff
@@ -1,5 +1,68 @@
 SecureHeaders::Configuration.default do |config|
-  config.csp = SecureHeaders::OPT_OUT
+  config.csp = {
+    default_src: [
+      "'self'", 
+      "https://*.quill.org",
+      "'unsafe-inline'"                                           # TODO: remove once nonce strategy is in place
+    ],                                                            # fallback for more specific directives
+
+    object_src: %w('none'),                                       # addresses <embed>, <object>, and <applet>
+
+    script_src: [
```

Comment:
So reviewing this is a bit tricky, but there are some frontend services that we use that aren't in this list (maybe they don't need to be, based on how they are used).

However, to be safe, it would be worth confirming if the following need to be added to one of these safelists:
- Intercom (support chat window)
- Coview (Intercom plugin to share browser info)
- Sentry (frontend error reporting)
- Clever Login (third-party login)
- Google Login (third-party login - may already be covered in the google urls)

Reply:
I added these items as a checklist at the top of the page.